src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (40 * days) - 20;
+  }
+
+  if (days >= 7) {
+    return (40 * days) - 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean

Reply:
Done)